projects/igniteui-angular/src/lib/paginator/paginator.component.ts

```diff
@@ -177,6 +178,38 @@ export class IgxPaginatorComponent extends DisplayDensityBase {
    @Input()
    public prepositionPage = CurrentResourceStrings.PaginatorResStrings.igx_paginator_pager_text;
 
+   /**
+    * An @Input property, sets a tooltip text for first page button label
+    * The default is 'Go to first page' localized string.
+    * @memberof IgxPaginatorComponent
+    */
+   @Input()
+   public firstPageLabel = CurrentResourceStrings.PaginatorResStrings.igx_paginator_first_page_button_text;
+
+   /**
+    * An @Input property, sets a tooltip text for the previous page button label
+    * The default is 'Previous page' localized string.
+    * @memberof IgxPaginatorComponent
+    */
+   @Input()
+   public previousPageLabel = CurrentResourceStrings.PaginatorResStrings.igx_paginator_previous_page_button_text;
+
+   /**
+    * An @Input property, sets a tooltip text for last page button label
+    * The default is 'Go to last page' localized string.
+    * @memberof IgxPaginatorComponent
+    */
+   @Input()
+   public lastPageLabel = CurrentResourceStrings.PaginatorResStrings.igx_paginator_last_page_button_text;
+
+   /**
+    * An @Input property, sets a tooltip text for the next page button label
+    * The default is 'Next page' localized string.
+    * @memberof IgxPaginatorComponent
+    */
+   @Input()
+   public nextPageLabel = CurrentResourceStrings.PaginatorResStrings.igx_paginator_next_page_button_text;
+
```

Comment:
In order to be consistent with the other components in the library, can you please expose a single property, to store all the necessary resource strings , rather to provide a 5 separate properties. You can see the igx-Carousel componet as a reference:
```
    @Input()
    set resourceStrings(value: IPaginatorResourceStrings) {
        this._resourceStrings = Object.assign({}, this._resourceStrings, value);
    }

    get resourceStrings(): IPaginatorResourceStrings{
        return this._resourceStrings;
    }
```